### ISSUE TITLE
Is it normal I dont get the PHPStorm Codecoverage feature to work ?

### ISSUE THREAD

**Issue by PhoenixFnX** [kenjis/ci-phpunit-test#69]
Hi kenjis, 

Can you have a look at this stackOverflow post, is it a ci-phpunit-test drawback ?

http://stackoverflow.com/questions/33146362/coverage-phpunit-0-with-jetbrains-phpstorm

**Comment by kenjis** [comment]
I don't know. What if you run phpunit from your terminal?

And if you use `master` branch, I added a bug yesterday, and I fixed now. I'm not sure the bug cause your error or not.

**Comment by PhoenixFnX** [comment]
Can you remind me how to switch to the master branch ?
Here is my composer : 

	"require-dev": {
		"kenjis/ci-phpunit-test": "^0.8.2"
	},

**Comment by PhoenixFnX** [comment]
Does this php coverage works with you ?
When I add this in the phpunit.xml file, it tells me Xdebug is not enabled:
```xml
	<logging>
		<log type="coverage-html" target="build/coverage"/>
		<log type="junit" target="build/logs/junit.xml" logIncompleteSkipped="false"/>
	</logging>
```

Therefore I tried enabling the xdebug debugger, the tests were veeeeeery slow to run and I got a HTML report in build/coverage folder. It's good but not integrated in PHPStorm environment.

So I went back to zend debugger and try the PHPStorm feature (removing the log section in phpunit.xml). Tests are quick to run but coverage keeps at 0%...

**Comment by kenjis** [comment]
If you want to use master branch, specify `"1.0.x@dev"`. But you don't have to use it in this case.

Yes, Xdebug is very slow.

I don't know PhpStorm.

Do  you follow the the below?
https://www.jetbrains.com/phpstorm/help/monitoring-code-coverage-for-php-applications.html

**Comment by PhoenixFnX** [comment]
Yes I do everything the tutorial says.
I have to be more specific and drilldown my configuration by writing a simple PHPUnit Test to see if your project is actually the problem... I will tell you when I get time to do that.

**Comment by kenjis** [comment]
Do you have the below in your phpunit.xml?

~~~
<logging>
    <log type="coverage-clover" target="build/logs/clover.xml"/>
</logging>
~~~

**Comment by PhoenixFnX** [comment]
well... I did many tests, here are my statements :

When I add a <log type= in my phpunit.xml, I had an error which tells
Xdebug must be installed. When I got that I tried to run the coverage
through PHPStorm but it doesn't work with zend debugger.

Then I changed my server debugger from Zend Debugger to Xdebug.

Now I have the coverage working adding the log in phpunit.xml. But it's
extremely long and I dont want it everytime.
Moreover, the report is not automatically included in the PHPStorm IDE.

Then I changed my PHPStorm setting to tell him I'm using XDebug and tadaaaa
the coverage button works !
It runs slowly but I can run it only when I want. And it IS integrated
through the IDE which is veeery useful !

Hope this helps.
Bye
